crates/e2e/tests/e2e/onchain_settlement.rs

```diff
@@ -1,170 +0,0 @@
-use {
-    e2e::{
-        setup::{colocation::SolverEngine, *},
-        tx,
-    },
-    ethcontract::prelude::U256,
-    model::{
-        order::{OrderCreation, OrderKind},
-        signature::EcdsaSigningScheme,
-    },
-    secp256k1::SecretKey,
-    shared::ethrpc::Web3,
-    web3::signing::SecretKeyRef,
-};
-
-#[tokio::test]
-#[ignore]
-async fn local_node_onchain_settlement() {
-    run_test(onchain_settlement).await;
-}
-
-async fn onchain_settlement(web3: Web3) {
-    let mut onchain = OnchainComponents::deploy(web3).await;
-
-    let [solver] = onchain.make_solvers(to_wei(1)).await;
-    let [trader_a, trader_b] = onchain.make_accounts(to_wei(1)).await;
-    let [token_a, token_b] = onchain
-        .deploy_tokens_with_weth_uni_v2_pools(to_wei(1_000), to_wei(1_000))
-        .await;
-
-    // Fund trader accounts
-    token_a.mint(trader_a.address(), to_wei(101)).await;
-    token_b.mint(trader_b.address(), to_wei(51)).await;
-
-    // Create and fund Uniswap pool
-    token_a.mint(solver.address(), to_wei(1000)).await;
-    token_b.mint(solver.address(), to_wei(1000)).await;
-    tx!(
-        solver.account(),
-        onchain
-            .contracts()
-            .uniswap_v2_factory
-            .create_pair(token_a.address(), token_b.address())
-    );
-    tx!(
-        solver.account(),
-        token_a.approve(
-            onchain.contracts().uniswap_v2_router.address(),
-            to_wei(1000)
-        )
-    );
-    tx!(
-        solver.account(),
-        token_b.approve(
-            onchain.contracts().uniswap_v2_router.address(),
-            to_wei(1000)
-        )
-    );
-    tx!(
-        solver.account(),
-        onchain.contracts().uniswap_v2_router.add_liquidity(
-            token_a.address(),
-            token_b.address(),
-            to_wei(1000),
-            to_wei(1000),
-            0_u64.into(),
-            0_u64.into(),
-            solver.address(),
-            U256::max_value(),
-        )
-    );
-
-    // Approve GPv2 for trading
-    tx!(
-        trader_a.account(),
-        token_a.approve(onchain.contracts().allowance, to_wei(101))
-    );
-    tx!(
-        trader_b.account(),
-        token_b.approve(onchain.contracts().allowance, to_wei(51))
-    );
-
-    let services = Services::new(onchain.contracts()).await;
-    let solver_endpoint = colocation::start_naive_solver().await;
```

Comment:
Why does this test have to be removed. I think it's one of few (maybe the only one?) that tests that the naive solver is actually able to settle batches.

Reply:
Naive solver does not support limit orders. When switched to using baseline (which does not do batch) I ended up with a classic limit order e2e test which is already covered with other tests.

Reply:
Ahh, right. Forgot that naive doesn't support computing the fee. :ok_hand:

Reply:
How hard would it be to teach the naive solver to compute fees? On a high level it should be less hard than what the baseline solver does since it only ever uses one kind of pool? I think it would be good to have some basic CoW test coverage.

Reply:
Can we capture a task to bring this test back if we resurrect naive solver (I do think for e2e tests specifically it would be good to have a solver that's capable of matching basic CoWs)